packages/ListBox/stories/examples/Keyboard/index.js

```diff
@@ -0,0 +1,333 @@
+      <Blocks>
+        <Block>
+          <h3>Single</h3>
+          <Card>
+            <ListBox>{characters.antiHeroes}</ListBox>
+          </Card>
+          <h3>Multi</h3>
+          <Card>
+            <ListBox isMulti>{characters.antiHeroes}</ListBox>
+          </Card>
+        </Block>
+        <Block>
+          <CheckListItem>
+            when trigger has focus and pressing <Up /> or <Down /> the popover menu opens
+          </CheckListItem>
+          <CheckListItem>
+            while the menu is open and there is not previous selection and is single selection, the component should
+            wait for an interaction from the user before selecting something either clicking or <Up />
+            <Down />. <br />
+            In case the ListBox has a multi prop enable, then it should focus on the first element if the component has
+            not selection.
+          </CheckListItem>
+          <CheckListItem>
+            it should be possible to navigate with <Up /> <Down /> across all the items
+          </CheckListItem>
+          <CheckListItem>
+            Once a value has been selected and the popover has been closed, when the user interacts again with the
+            ListBox and the popover has been opened, the focus should focus on the latest selected item if has the
+            single selection prop on. Meanwhile if has isMulti on should just open the ListBox.
+            <br />
+            <br />
+          </CheckListItem>
+          <CheckListItem>
+            If the user reaches the top or the bottom length of the List, the focus should remain in the first or last
+            item depending the cases. We are not looping the list to avoid confusion to our users.
+          </CheckListItem>
+        </Block>
+      </Blocks>
+    </>
+  );
+}
+
+export function WithFilterStory() {
+  return (
+    <>
+      <h2>Case with filters</h2>
+      <Blocks>
+        <Block>
+          <h3>Single</h3>
+          <Card>
+            <ListBox>
+              <ListBox.Filter />
+              {characters.antiHeroes}
+            </ListBox>
+          </Card>
+          <h3>Multi</h3>
+          <Card>
+            <ListBox isMulti>
+              <ListBox.Filter />
+              {characters.antiHeroes}
+            </ListBox>
+          </Card>
+        </Block>
+        <Block>
+          <Toast hasCloseButton={false}>
+            The keyboard navigation should has the same behaviour as the previous examples with few variations, since
+            now includes a filter in both examples.
+          </Toast>
+          <CheckListItem>
+            When there is not an item selected and ListBox hasFilter prop on, it should focus automatically on the
+            filter input after the popover opened, this applies to both case isMulti and single selection
+          </CheckListItem>
+          <CheckListItem>
+            Once the ListBox has at least one selected option, it should focus on it even if the filter exists.
```

Comment:
Needs a little adjustment – this describes only the single select version.

Reply:
There is no action for the multiselect only open the `box`

Reply:
Exactly!  But this suggests it should focus on on the selected option.  Just needs a little rewording.